### DISCUSSION TITLE
Feature Request: Import Directly From OpenAI

### DISCUSSION THREAD

**Discussion by Ptolemy2002** [ztjhz/BetterChatGPT#429] [Ideas]
Currently, there is only an option to import chat messages from a JSON format. OpenAI has a feature that allows users to view a snapshot of chat conversations at a specific point in time. I'm wondering if it is possible to input one of those links into the app and have it automatically copy the data. The default config could be used, and a new ID could be generated.

**Comment by RedwindA** [comment]
https://greasyfork.org/en/scripts/456055-chatgpt-exporter

**Reply by Ptolemy2002** [reply]
This script seems to output the chats in an unrecognized format for the app, but it is relatively easy to convert. I wrote a basic script for that at https://pastebin.com/gmiEGLVe

**Reply by RedwindA** [reply]
thank you for this useful script❤
